documents/refund.md

```diff
@@ -141,8 +152,12 @@ instance.payments.fetch_multiple_refund(paymentId,option)
 -------------------------------------------------------------------------------------------------------
 
 ### Fetch a specific refund for a payment
-```js
-instance.payments.fetch(paymentId,refundId)
+```rb
+paymentId = "pay_FIKOnlyii5QGNx"
+
+refundId = "rfnd_FP8DDKxqJif6ca"
+
+Razorpay::Payment.fetch(paymentId).fetch_refund(refundId)
```

Comment:
Where is this declared?

Reply:
its declared in payment module 
https://github.com/razorpay/razorpay-ruby/blob/payment_update/lib/razorpay/payment.rb#L38

Reply:
Part of separate branch. Pls describe this function in the correct place.

Reply:
according to [doc](https://razorpay.com/docs/api/refunds/#fetch-a-specific-refund-for-a-payment) this function is belongs to refund module that why i have added this function in refund.md